pkg/otlp/internal/serializerexporter/consumer.go

```diff
@@ -87,7 +87,7 @@ func (c *serializerConsumer) ConsumeTimeSeries(ctx context.Context, dimensions *
 			Tags:     tagset.CompositeTagsFromSlice(c.enrichedTags(dimensions)),
 			Host:     dimensions.Host(),
 			MType:    apiTypeFromTranslatorType(typ),
-			Interval: 1,
+			Interval: 0, // OTLP metrics do not have an interval.
```

Comment:
Is setting the `Interval` to zero the same as not setting an interval for the serializer?